pkg/forwarder/forwarder_health.go

```diff
@@ -103,10 +103,7 @@ func (fh *forwarderHealth) healthCheckLoop() {
 				return
 			}
 		case <-fh.health.C:
-			if transactionsDroppedOnInput.Value() != 0 {
-				log.Errorf("Detected dropped transaction, reporting the forwarder as unhealthy: %v.", transactionsDroppedOnInput)
-				return
-			}
+			continue
```

Comment:
Leaving this empty is equivalent to the continue here. Not sure what I prefer, but we usually leave it empty (search for `health.C:`).

Reply:
Will leave it empty for consistency.